Content/Items/Forest/Armors.SlimePrince.cs

```diff
@@ -43,6 +43,13 @@ public override void SetDefaults()
 			Item.defense = 2;
 		}
 
+		public override ModItem Clone(Item newEntity)
+		{
+			var clone = base.Clone(newEntity) as SlimePrinceHead;
+			clone.prince = null;
```

Comment:
This may not be necessary. Looking at `PoltergeistArmor` which clones the list of minions, you can just annotate `prince` with `[CloneByReference]`.

I'm assuming that these sorts of items have their `Projectile` reference cleared (or otherwise invalidated) when unequipped, so there's not really any problem cloning the `null` (or dead) references.